src/com/google/cose/Ec2SigningKey.java

```diff
@@ -114,6 +118,74 @@ public ECPublicKey getPublicKey() {
     return (ECPublicKey) this.keyPair.getPublic();
   }
 
+  private static byte[] arrayFromBigNum(BigInteger n, int curveSize) {
```

Comment:
nit: please indicate this is always big-endian somewhere

Maybe even in the method name?

We should be careful so folks know not to ever try and reuse this for Ed25519 or Ed448 (who need LE encoding, IIUC).

Reply:
Added a comment indicating Big Endian on method, done.